pages/common/exenv.md

```diff
@@ -0,0 +1,24 @@
+# exenv
+
+> A tool to easily install Elixir versions and manage application environments.
+> More information: <https://github.com/exenv/exenv>.
+
+- Display a list of installed versions:
+
+`exenv versions`
+
+- Use a specific version of Elixir across the whole system:
+
+`exenv global {{version}}`
+
+- Use a specific version of Elixir for an application/project directory:
```

Comment:
Perhaps this would be more precise on what it actually does:
```suggestion
- Use a specific version of Elixir for the current application/project directory:
```